great_expectations/render/renderer/column_section_renderer.py

```diff
@@ -107,62 +200,248 @@ def _render_overview_table(cls, evrs, content_blocks):
+                "header": "Statistics",
+                "table_rows": table_rows,
+                "styling": {
+                    "classes": ["col-4"],
+                    "styles": {
+                        "margin-top": "20px"
+                    },
+                    "body": {
+                        "classes": ["table", "table-sm", "table-unbordered"],
+                    }
+                },
+            })
+        else:
+            return
 
     @classmethod
     def _render_values_set(cls, evrs, content_blocks):
-        # FIXME: Eugene, here's where the logic for using "cardinality" in form of which expectations are defined
-        # should determine which blocks are generated
-        # Relatedly, this will change to grab values_list and to use expect_column_distinct_values_to_be_in_set
         set_evr = cls._find_evr_by_type(
             evrs,
-            "expect_column_distinct_values_to_be_in_set"
+            "expect_column_values_to_be_in_set"
         )
 
+        # FIXME: This logic is very brittle. It will work on profiled EVRs, but not much else.
         if set_evr and "partial_unexpected_counts" in set_evr["result"]:
             result_key = "partial_unexpected_counts"
         elif set_evr and "partial_unexpected_list" in set_evr["result"]:
             result_key = "partial_unexpected_list"
         else:
             return
 
-        if len(set_evr["result"][result_key]) < 10:
-            content_blocks.append(
-                ValueListContentBlockRenderer.render(
-                    set_evr,
-                    result_key=result_key
-                )
-            )
+        partial_unexpected_counts = set_evr["result"]["partial_unexpected_counts"]
+        values = [str(v["value"]) for v in partial_unexpected_counts]
+
+        if len(" ".join(values)) > 100:
+            classes = ["col-12"]
         else:
-            content_blocks.append(
-                GraphContentBlockRenderer.render(
-                    set_evr,
-                    result_key=result_key
-                )
-            )
+            classes = ["col-4"]
+
+        # TODO: This approach to styling is way too complicated for a simple values lists.
+        new_block = {
+            "content_block_type": "value_list",
+            "header": "Example values",
+            "value_list": [{
+                "template": "$value",
+                "params": {
+                    "value": value
+                },
+                "styling": {
+                    "default": {
+                        "classes": ["badge", "badge-info"]
+                    }
+                }
+            } for value in values],
+            "styling": {
+                "classes": classes,
+                "styles": {
+                    "margin-top": "20px",
+                }
+            }
+        }
+        # print(new_block)
+
+        content_blocks.append(new_block)
+
+        # if len(set_evr["result"][result_key]) < 10:
```

Comment:
Ready to axe this? Is GraphContentBlock used anymore (i.e. can we delete the entire file?)

Reply:
Not sure. Don't have full context on state since I jumped in near end. @abegong ^^

Reply:
Actually, yeah. It looks like this can be axed.